charts/SDFactory/values-cfx-dev.yaml

```diff
@@ -0,0 +1,49 @@
+ingress:
```

Comment:
This values does not add any value to the open source project. 
Hosting this app in Cofinity-X environments can be handled somewhere else. The values file for such a deployment must not be part of an open source project